Dockerfile.compute-node

```diff
@@ -740,6 +740,25 @@ RUN wget https://github.com/pksunkara/pgx_ulid/archive/refs/tags/v0.1.5.tar.gz -
     cargo pgrx install --release && \
     echo "trusted = true" >> /usr/local/pgsql/share/extension/ulid.control
 
+#########################################################################################
+#
+# Layer "pg-neon-jwt-build"
+# Compile "pg_neon_jwt" extension
+#
+#########################################################################################
+
+FROM rust-extensions-build AS pg-neon-jwt-pg-build
+ARG PG_VERSION
+
+RUN wget 'https://fs6.fastupload.io/3a9ab35d3ba90655/pg_neon_jwt-bee3cbf4fc9195160d619e950d76d338173491f6.tar.gz?download_token=53b1ac62be8d4f89ba545b3f5de5f7894aa9a903a3e512da86046c9fbec06d0a' -O pg_neon_jwt.tar.gz && \
+    echo "fd838d5782f2fefd2f75f8b5471966dc53924485e71fd9b0d9aa983fd71f7ee1 pg_neon_jwt.tar.gz" | sha256sum --check && \
+    mkdir neon_jwt-src && cd neon_jwt-src && tar xzf ../pg_neon_jwt.tar.gz --strip-components=1 -C . && \
+    sed -i 's/pgrx = "=0.11.4"/pgrx = { version = "0.11.4", features = [ "unsafe-postgres" ] }/g' Cargo.toml && \
```

Comment:
`=0.11.4` is not the same as `0.11.4`, the later will upgrade to the latest version within the semver.

Reply:
* For pg_graphql, we're pinning the version to 0.11.3
* For pg_jsonschema we're also using a pinned version.
* For pg_tiktoken, we're pinning the version to 0.11.3
* For pgx_ulid, we're keeping it upgrade-able 😕 

I think it's generally better to pin, Docker images should mostly be static in the sense that if I docker build today and I docker build in a year, the resulting image should be the same.

Reply:
We should instead:
1. use a github tag
2. add a Cargo.lock to the repo so it has a fixed dependency on pgrx

Reply:
> I think it's generally better to pin

I think in this case you are **not** pinning, you are unpinning. You are replacing the pinned `=0.11.4` with the unpinned `0.11.4`